src/renderer/components/DeviceAction/index.js

```diff
@@ -32,7 +32,7 @@ import {
   renderRequiresAppInstallation,
   renderListingApps,
   renderWarningOutdated,
-  renderSwapDeviceConfirmationV2,
+  renderSwapDeviceConfirmation,
```

Comment:
The fact that you are changing back to the old rendering concerns me. The `renderSwapDeviceConfirmation` that was developed for the original swap almost two years ago was replaced by the work from @elbywan late last year. I ask you to sync with him and actually finally drop any remaining code from the v1 of the feature to avoid problems such as this one 🙏🏼

Reply:
`renderSwapDeviceConfirmation` was not used anywhere and was dead code. I just renamed `renderSwapDeviceConfirmationV2` to `renderSwapDeviceConfirmation` and cleanup the code. I'm not using the old one.

Reply:
LGTM 👍 , as far as I can see the export has been renamed to remove the `V2` suffix but the code seems to be the correct one